answers/src/main/scala/fpinscala/laziness/Stream.scala

```diff
@@ -1,4 +1,6 @@
 package fpinscala.laziness
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
```

Comment:
I don't think you meant to add those imports? They break compilation, scalatest isn't available for the main compilation unit, unless for the tests.